src/index.html

```diff
@@ -11,7 +11,143 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```
Please fix all cases
![image](https://user-images.githubusercontent.com/96409563/206401585-86e6b7f9-9908-49ac-b0ef-5f613ecb8301.png)